test/components/baseAxisTests.ts

```diff
@@ -196,16 +196,4 @@ describe("BaseAxis", () => {
 
     svg.remove();
   });
-
-  it("default alignment based on orientation", () => {
```

Comment:
Based on the method shuffling, since the default alignment is determined at the anchor stage (I'm trying to go with a convention of things that affect visual details should be done at the anchor step), it's not determined at the constructor step.

We could have this test still exist by setting the yAlignProportion early... I'm not sure if this is the best place to put the setting.

Also, I guess another aspect is that this touches underscored instance variables, which in my mind, means it's a lot more prone to change from source code changes easily to the point of being outdated